### ISSUE TITLE
Any running jobs fail to resume after Jenkins restart with WorkingHours plugin installed

### ISSUE THREAD

**Issue by faucherb94** [jenkinsci/working-hours-plugin#35]
When the WorkingHours plugin is installed, any jobs that have it enabled via the `enforceBuildSchedule()` option that were running at the time of a restart, will fail to resume after Jenkins restarts, showing the following error repeatedly in their log:

```
Waiting to resume part of <job> » <job>:<run>: Exception evaluating if the queue can run the task
```

The Jenkins log shows this stacktrace:

```
Exception evaluating if the queue can run the task 'part of <job>'
java.lang.NullPointerException
	at org.jenkinsci.plugins.workinghours.WorkingHoursQueueTaskDispatcher.canRunNow(WorkingHoursQueueTaskDispatcher.java:120)
	at org.jenkinsci.plugins.workinghours.WorkingHoursQueueTaskDispatcher.canRun(WorkingHoursQueueTaskDispatcher.java:82)
	at hudson.model.Queue.getCauseOfBlockageForItem(Queue.java:1209)
	at hudson.model.Queue.maintain(Queue.java:1539)
	at hudson.model.Queue$MaintainTask.doRun(Queue.java:2904)
	at hudson.triggers.SafeTimerTask.run(SafeTimerTask.java:91)
	at jenkins.security.ImpersonatingScheduledExecutorService$1.run(ImpersonatingScheduledExecutorService.java:58)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

The only resolution is aborting and restarting the jobs.

System info:

Jenkins version: 2.249.1 (seen for several of the past LTS versions)
WorkingHours Plugin version: 1.0

**Comment by kokorin** [comment]
We found the same behavior.  I noticed that code in master branch doesn't correspond to the latest published plugin version: 1.0. 
Master branch has no executable code at `WorkingHoursQueueTaskDispatcher.java:120`.

**Comment by Marc43** [comment]
I would like to use this plugin, this behaviour seems too risky to install the plugin. Waiting until the developers say something about it.

**Comment by jeffpearce** [comment]
Is the repro just

> - Start a job
> - Restart Jenkins

I can take a look and see what's going on
I can take a look and see if I can figure out how to

**Comment by faucherb94** [comment]
> Is the repro just
> 
> > * Start a job
> > * Restart Jenkins
> 
> I can take a look and see what's going on

Start a job with `enforceBuildSchedule` enabled.

**Comment by jeffpearce** [comment]
Thanks for reporting - I just released 1.1 to fix this. Please reopen the issue if you still see it
